tests/testthat/test-carehomes.R

```diff
@@ -12,16 +12,15 @@ test_that("can run the carehomes model", {
   res <- mod$run(end)
 
   expected <- rbind(
-    icu = c(4, 26, 2, 13, 6),
-    general = c(32, 72, 11, 37, 33),
+    icu = c(4, 25, 2, 13, 5),
+    general = c(31, 71, 11, 38, 34),
     deaths_comm = c(15825, 15971, 15955, 15914, 15882),
-    deaths_hosp = c(305577, 305933, 305954, 305572, 306014),
-    deaths_tot = c(321401, 321903, 321909, 321486, 321896),
+    deaths_hosp = c(305578, 305933, 305954, 305572, 306014),
     admitted = c(151349, 150757, 151137, 150569, 150964),
     new = c(483755, 484827, 484942, 483498, 483546),
-    R_pre_15_64 = c(2033, 5320, 932, 3088, 2122),
-    R_neg_15_64 = c(5470674, 5472544, 5474482, 5473012, 5475747),
-    R_pos_15_64 = c(25149548, 25144489, 25147586, 25147430, 25145179))
+    prob_pos = c(0.7239, 0.7238, 0.7239, 0.7238, 0.7238))
+  #round prob_pos in res so they should match above
+  res[seq(7, 35, 7)] <- round(res[seq(7, 35, 7)], 4)
```

Comment:
```suggestion
    prob_pos = c(0.723911065300095, 0.723769622060999, 0.723852644013753, 
                 0.723847861727815, 0.723785265474777))
```

This was generated with `dput()` and should match exactly